src/services/ldap/strategies/iserv-idm.js

```diff
@@ -31,7 +31,7 @@ class IservIdmLDAPStrategy extends AbstractLDAPStrategy {
 	async getUsers(school) {
 		const requiredAttributes = '(objectClass=person)(sn=*)(uuid=*)(uid=*)(mail=*)(cn=*)';
 		const options = {
-			filter: filterForModifiedEntities(this.config.lastModifyTimestamp, `(&${requiredAttributes})`),
+			filter: filterForModifiedEntities(this.getLastModifiedTimestamp(this.config), `(&${requiredAttributes})`),
```

Comment:
this is disabling forceFullSync which just depends on having no lastModifyTimestamp defined

Reply:
Okay, then how a full sync is force are not good implemented.
A full sync shoudn't deppend on if there is a lastModifyTimestamp or not.
If i unserstend right, if the filed lastModifyTimestamp is deleted once it will be set to 0 on success or are deleted for ever at a fail casse.